packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -30,15 +30,25 @@ interface WatchPR {
   url: string;
 }
 
-handler.listPRs = async function listPRs(): Promise<WatchPR[]> {
+handler.getDatastore = async function getDatastore() {
   const query = datastore.createQuery(TABLE).order('created');
   const [prs] = await datastore.runQuery(query);
+  return [prs];
+};
+
+handler.listPRs = async function listPRs(): Promise<WatchPR[]> {
+  const [prs] = await handler.getDatastore();
```

Comment:
Yeah, if we're just destructuring the array here, no reason to return an array of results above.  We can remove the `[` and `]` here, and in the function that returns the results.

Reply:
Thanks for taking a look, Justin! To be honest, I had some trouble with this function. We're technically returning [this](https://cloud.google.com/datastore/docs/reference/data/rpc/google.datastore.v1#google.datastore.v1.RunQueryResponse
) type, but it's actually just the QueryResultBatch part of that Response, since we're selecting an array. However, that data type is not exported, and TS won't let me convert it to my own type. I'm probably doing something wrong, I just can't quite figure out what it is! :)